### PR TITLE
MultiQC v1.4 release

### PR DESCRIPTION
**Pull request by ewels**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

NB: Will currently fail due to old versions of `colormath` and `spectra` on conda-forge.

* Colormath conda-forge PR: https://github.com/conda-forge/colormath-feedstock/pull/2 _(merged)_
* Spectra conda-forge PR: https://github.com/conda-forge/spectra-feedstock/pull/1

### CODE REVIEW

**Comment by epruesse** [comment]
This is waiting on conda-forge/spectra-feedstock#1

File: recipes/multiqc/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  md5: 67c4aae8b85ae40fcb2e98319fb629ac
+  fn: v1.4.tar.gz
+  url: https://github.com/ewels/MultiQC/archive/v1.4.tar.gz
+  md5: 83a20323d7b72b588b5e2c5e5074056b
   patches:
     - setup.patch # [py3k]
 
 build:
-  number: 2
+  number: 1
```
Comment (mbargull) [inline recipes/multiqc/meta.yaml]: This should be
```yaml
  number: 0
```

File: recipes/multiqc/meta.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-  version: "1.3"
+  version: "1.4"
 
 source:
-  fn: v1.3.tar.gz
-  url: https://github.com/ewels/MultiQC/archive/v1.3.tar.gz
-  md5: 67c4aae8b85ae40fcb2e98319fb629ac
+  fn: v1.4.tar.gz
+  url: https://github.com/ewels/MultiQC/archive/v1.4.tar.gz
+  md5: 83a20323d7b72b588b5e2c5e5074056b
```
Comment (mbargull) [inline recipes/multiqc/meta.yaml]: For checksums that aren't provided alongside the source, we should prefer `sha256` (http://bioconda.github.io/guidelines.html#hashes).

File: recipes/multiqc/meta.yaml [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
   patches:
     - setup.patch # [py3k]
```
Comment (mbargull) [inline recipes/multiqc/meta.yaml]: Since the patch is removed,
```diff
-  patches:
-    - setup.patch # [py3k]
```

**Comment by ewels** [comment]
Thanks for the review @mbargull 👍 Changes pushed..

**Comment by ewels** [comment]
> **Degraded performance for all Mac builds**
https://www.traviscistatus.com/incidents/mc9x2wmpnvhg

**Comment by ewels** [comment]
Crossref https://github.com/bioconda/bioconda-recipes/pull/7314#issuecomment-358977980 - is it worth waiting to merge this v1.4 release until we patch v1.3, so that we have a working version of v1.3 for people wanting to use that version in the future?

**Comment by dpryan79** [comment]
If you want to patch 1.3 then you can just put it in a subdirectory (see the deeptools recipe as an example). Then you can have your cake and merge 1.4 too.

**Comment by ewels** [comment]
Yeah I saw that - just seems a bit messy / kind of overkill to copy all of the code just to add ~5 characters somehow 😛

**Comment by daler** [comment]
xref https://github.com/bioconda/bioconda-recipes/pull/7314#issuecomment-358991449, v1.3 before v1.4 should be fine to avoid clutter in the directory. The disadvantage is the currently-very-long wait time on travis for separate PRs.

**Comment by epruesse** [comment]
You could always delete the 1.3 directory again after it's built. Not pretty either, but should be fine. +1 for fixing the 1.3 (or else marking it broken).

**Comment by mbargull** [comment]
1.3 should already work without changes now:
```bash
$ conda create --dry-run -nx multiqc=1.3|grep 'colormath\|multiqc\|networkx\|spectra'
    colormath:                     2.1.1-py36_1     conda-forge
    multiqc:                       1.3-py36_2       bioconda   
    networkx:                      1.11-py36_0      conda-forge
    spectra:                       0.0.7-py36_0     conda-forge
```

**Comment by ewels** [comment]
Ok great - I think we're getting there! v1.3 working and I've just removed `networkx` as MultiQC requirement as it should all be handled normally now that `spectra` and `colormath` have the correct requirements themselves.

Hopefully now just a case of waiting for the Travis run to complete, then merging 👍 

Phil

**Comment by chapmanb** [comment]
Phil and Marcel -- thank you for all the hard work on this. I've been patiently waiting on this through the OSX backup so am psyched to get in merged. Much appreciated.

**Comment by ewels** [comment]
Ah brilliant, it finally went through! Thanks for the merge @chapmanb 👍

**Comment by ewels** [comment]
And yes, thanks for all of the help @mbargull and everyone else! Much appreciated.

**Pull request merged**